### PR TITLE
Fixed all Places I expected/found errors

### PR DESCRIPTION
**Pull request by greysilly7**
Hi I am back, I went through the typing cause I encountered more errors and others I found in it, so I fixed it should work

### CODE REVIEW

File: typings/index.d.ts [hunk 1]
```diff
@@ -15,23 +15,23 @@ interface PlayerEvents {
...
+    sortBy?: 'relevance'|'date'|'view count'|'rating',
+    requestedBy?: String,
+    index?: Number
 }
 type PlaylistOptions = {
-    search: String,
-    maxSongs: Number,
-    requestedBy: String
-    shuffle: Boolean,
+    search?: String,
```
Comment (SushiBtw) [inline typings/index.d.ts:26]: search is required in PlaylistOptions

File: typings/index.d.ts [hunk 2]
```diff
@@ -15,23 +15,23 @@ interface PlayerEvents {
     clientUndeafen: [message: Message, queue: Queue]
 }
 type PlayOptions = {
-    search: String,
-    uploadDate: 'hour'|'today'|'week'|'month'|'year',
-    duration: 'short'|'long',
-    sortBy: 'relevance'|'date'|'view count'|'rating',
-    requestedBy: String,
-    index: Number
+    search?: String,
```
Comment (SushiBtw) [inline typings/index.d.ts:18]: search is required in PlayOptions

**Review by SushiBtw — COMMENTED** [review]
can you please remove the ? from required values?

**Comment by greysilly7** [comment]
Yeah

**Comment by SushiBtw** [comment]
thanks

**Pull request merged**